libs/parsers/src/lib/expression/parser.ts

```diff
@@ -43,14 +50,14 @@ export class ExpressionParser {
     },
   ];
 
-  public static parseExpression(expression: string): Expression {
-    const scanner = new ExpressionScanner(expression);
-    const parsedExpression = ExpressionParser._parseExpressionRecursively(scanner);
+  public static parseExpression(expression: string, isAliasPathParsingEnabled?: boolean): Expression {
+    const scanner = new ExpressionScanner(expression, /*prefetch*/ true);
```

Comment:
You do not need to set prefetch here explicitly since it is already a default set value to true in the constructor